content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,236 @@
+
+As a contributor, Matt should be able to provide all the following details.
+
+* **ID:** A workshop ID to be used by backend server automation and Replays table to reference the associated replay video of the workshop (automatically created at the import of the wod.yml file process)
+* **name:** The workshop's name as it will will be displayed on the registration portal
+* **notebook:** Name of the folder containing all the workshop's notebooks (automatically created at the import of the wod.yml file process)
+* **description:** The workshop's abstract as it will will be displayed on the registration portal
+* **avatar, role and replayLink** are superseded by entries in the replay table. I will explain later
+* **R﻿eplayId:** This entry links the dedicated replay video to the workshop. it enables the presence of the replay in the learn more page of the workshop (automatically created at the import of the wod.yml file process)
+* **Category:** The workshops' registration portal proposes several filters to display the catlog's content. You can view all workshops, the most poular ones, or by category. Use this field to sort workshops accordingly
+* **D﻿uration:** All workshops are time bombed. You will define here the time allocated to perform the workshop
+* **A﻿ctive:** Tag to set to enable visibility of the workshop's tile in the registration portal
+* **W﻿orkshopImg:** As part of the lifecycle of the workshop, several emails are sent to the student. A workshop image is embedded in the first emails
+* **session type:** Workshops-on-Demand by default (automatically created at the import of the wod.yml file process)
+
+The following fields are required by the infrastructure. And I will work as the infrastructure Admin with Matt to define them.
+
+* **capacity:** The number of maximum concurrent students allowed to take on the workshop
+* **range:** The range between which students get picked at registration time
+* **r﻿eset and ldap** entries are to be used by backend server automation if dedicated reset scripts and ldap authentication are required by the workshop
+* **location:** If your setup includes multiple jupyterhub servers, use this field to allocate workshops according to your needs.
+* **compile:** This entry will be filled with the name of a script to be compiled at deployment time. This feature allows the admin to hide login scripts and credentials in non-editable executable files
+* **varpass:**  This defines whether or not a workshop requires a password variable needs to be leveraged
+* **B﻿adgeImg:** As part of the lifecycle of the workshop, several emails are sent to the student. In the final email, a badge is included. It allows the student to share its accomplishment on social media like linkedin for instance
+* **beta:** Not implemented yet :-)
+* **AlternateLocation:** future development. The purpose is to allow automation of the relocation of a workshop in case of primary location's failure
```

Comment:
I would have done this one this way:

AlternateLocation: (Future development) The purpose is to allow automation of the relocation of a workshop in case of primary location's failure